crates/turborepo-paths/src/absolute_system_path_buf.rs

```diff
@@ -202,6 +202,10 @@ impl AbsoluteSystemPathBuf {
         fs::remove_file(self.0.as_path())
     }
 
+    pub fn remove_dir_all(&self) -> Result<(), io::Error> {
```

Comment:
This should be implemented on `AbsoluteSystemPath` instead of the owned type

Reply:
Good call. We should probably clean that up in general, there's currently a mix